config/crd/cluster.open-cluster-management.io_hypershiftdeployments.yaml

```diff
@@ -103,44 +103,44 @@ spec:
                   by ACM, if omitted, it will be generated
                 properties:
                   additionalTrustBundle:
-                    description: AdditionalTrustBundle is a reference to a ConfigMap
-                      containing a PEM-encoded X.509 certificate bundle that will
-                      be added to the hosted controlplane and nodes
+                    description: AdditionalTrustBundle is a reference to a ConfigMap containing
```

Comment:
once this CRD is updated, please remember to update the CRD in backplane-operator.

Reply:
Yes the CRDs are updated in backplane-operator